<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="palette">
        <palette>
         <active/>
         <inactive/>
         <disabled/>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true">margin: 20px;
margin-left: 30px;
margin-top: 10px;</string>
       </property>
       <property name="title">
        <string>Xmin:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEditXmin">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
  background-color: rgb(114, 159, 207);
  border-radius: 3px;
}

:hover {
  border: 1px solid #006080;
  color: black;
  font: 25 italic 12pt &quot;Ubuntu&quot;;
  background-color: rgb(211, 215, 207);
}
</string>
          </property>
          <property name="text">
           <string>0.1</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="styleSheet">
        <string notr="true">margin: 20px;
margin-right: 30px;
margin-top: 10px;</string>
       </property>
       <property name="title">
        <string>Xmax:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lineEditXmax">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
  background-color: rgb(114, 159, 207);
  border-radius: 3px;
}

:hover {
  border: 1px solid #006080;
  color: black;
  font: 25 italic 12pt &quot;Ubuntu&quot;;
  background-color: rgb(211, 215, 207);
}
</string>
          </property>
          <property name="text">
           <string>100.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="styleSheet">
        <string notr="true">margin: 20px;
margin-left: 30px;
margin-top: 10px;</string>
       </property>
       <property name="title">
        <string>Ymin:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEditYmin">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
  background-color: rgb(114, 159, 207);
  border-radius: 3px;
}

:hover {
  border: 1px solid #006080;
  color: black;
  font: 25 italic 12pt &quot;Ubuntu&quot;;
  background-color: rgb(211, 215, 207);
}
</string>
          </property>
          <property name="text">
           <string>-100.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="styleSheet">
        <string notr="true">margin: 20px;
margin-right: 30px;
margin-top: 10px;</string>
       </property>
       <property name="title">
        <string>Ymax:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignHCenter</set>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEditYmax">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
  background-color: rgb(114, 159, 207);
  border-radius: 3px;
}

:hover {
  border: 1px solid #006080;
  color: black;
  font: 25 italic 12pt &quot;Ubuntu&quot;;
  background-color: rgb(211, 215, 207);
}
</string>
          </property>
          <property name="text">
           <string>100.0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="ApplySettingsButton">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>35</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#ApplySettingsButton {
  background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 255), stop:0.09 rgba(246, 189, 237, 255), stop:0.14 rgba(194, 207, 246, 255), stop:0.19 rgba(184, 160, 168, 255), stop:0.25 rgba(171, 186, 248, 255), stop:0.32 rgba(243, 248, 224, 255), stop:0.385 rgba(249, 162, 183, 255), stop:0.47 rgba(100, 115, 124, 255), stop:0.58 rgba(251, 205, 202, 255), stop:0.65 rgba(170, 128, 185, 255), stop:0.75 rgba(252, 222, 204, 255), stop:0.805 rgba(206, 122, 218, 255), stop:0.86 rgba(254, 223, 175, 255), stop:0.91 rgba(254, 236, 244, 255), stop:1 rgba(255, 191, 221, 255));
  border-radius: 10px;
}

QPushButton#ApplySettingsButton:hover {
  background-color: rgb(239, 41, 41);
  border: 1px solid black;
}</string>
     </property>
     <property name="text">
      <string>Apply settings</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
